cmd/object-api-getobject_test.go

```diff
@@ -159,6 +159,7 @@ func testGetObject(obj ObjectLayer, instanceType string, t TestErrHandler) {
 		if err != nil && !testCase.shouldPass {
 			if !strings.Contains(err.Error(), testCase.err.Error()) {
 				t.Errorf("Test %d: %s: Expected to fail with error \"%s\", but instead failed with error \"%s\" instead.", i+1, instanceType, testCase.err.Error(), err.Error())
+				errorIf(err, "error")
```

Comment:
i had added `errorIf(err, "error")` to get the stack trace to see where the error was originating, will remove it